src/WebApiTestApplication/Watts.config.json

```diff
@@ -1,41 +1,42 @@
 ﻿{
-    "WebApiModuleFileName": "./bin/WebApiTestApplication.dll",
+  "WebApiModuleFileName": "./bin/WebApiTestApplication.dll",
 
-    "EndpointsOutputDirectory": "./Scripts/Endpoints",
-    "EndpointsSupportCaching": true,
-    "EndpointsFileName": "Endpoints.ts",
-    "EndpointsNamespace": "Endpoints",
+  "GenerateEndpointsReturnTypes": true,
 
-    "GenerateService": true,
-    "ServiceOutputDirectory": "./Scripts/Endpoints",
-    "ServiceFileName": "Service.ts",
-    "ServiceNamespace": "Endpoints",
+  "EndpointsOutputDirectory": "./Scripts/Endpoints",
+  "EndpointsSupportCaching": true,
+  "EndpointsFileName": "Endpoints.ts",
+  "EndpointsNamespace": "Endpoints",
 
-    "GenerateEnums": true,
-    "GenerateEnumDescriptions":  true, 
-    "EnumsOutputDirectory": "./Scripts/Enums",
-    "EnumsFileName": "Enums.ts",
-    "EnumsNamespace": "Enums",
+  "GenerateService": true,
+  "ServiceOutputDirectory": "./Scripts/Endpoints",
+  "ServiceFileName": "Service.ts",
+  "ServiceNamespace": "Endpoints",
 
-    "GenerateInterfaces": true,
-    "InterfacesOutputDirectory": "./Scripts/Interfaces",
-    "InterfacesFileName": "Interfaces.ts",
-    "InterfacesNamespace": "Interfaces",
-    "InterfaceMatches": [
-        {
-            "Match": "WebApiTestApplication.Controllers.*Chain.*$"
```

Comment:
Return these please. I thought you were modifying some other config file, not the webapitestapplication one :(
Sorry about that!